app/views/partials/inputSingleCheckbox.scala.html

```diff
@@ -13,7 +13,9 @@
 @isSmallCheckbox = {@attributes.get(Symbol("_smallCheckbox"))}
 @isDynamicForm = @{attributes.getOrElse(Symbol("_dynamicForm"), false).asInstanceOf[Boolean]}
 @fieldId = @{if(isDynamicForm) s"inputcheckbox-$fieldName" else fieldName}
-    <div class="govuk-checkboxes @{if(fieldError.nonEmpty) "govuk-form-group--error" else ""}">
+    @if(!isDynamicForm) {
```

Comment:
I'd avoid splitting the HTML up like this using `if` statements. It gets confusing to read. Probably better to keep it all in one `if` statement if that is possible, eg:
```
...
@if(!isDynamicForm) { 
   .... all the HTML needed for this ...
} else { 
   ... all the HTML for the default ...
}
...
```

Reply:
This makes sense. I've extracted out the common html and put everything into a single if else statement.